src/components/header/renderer.tsx

```diff
@@ -71,6 +72,30 @@ class Header extends React.Component<Props, State> {
     });
   }
 
+  public handleHelpModalToggle(event, openPortal, closePortal, isOpen) {
+    if (this._listnerAttached) {
+      return;
+    }
+
+    window.addEventListener('keydown', event => {
+      if (
+        (event.keyCode === 222 && event.metaKey && !event.shiftKey) || // Handle key press in Mac
```

Comment:
Will be better if we store these key codes in `consts.ts`

Reply:
Yes! I will do it.

Reply:
This is still todo, right?

Reply:
@siddhant1 fix more of these types of changes in workbench cleanup. It's okay if this is resolved here itself.